apiserver/apiserver.go

```diff
@@ -648,10 +648,13 @@ func (srv *Server) loop(ready chan struct{}) error {
 	return tomb.ErrDying
 }
 
-func (srv *Server) endpoints() ([]apihttp.Endpoint, error) {
-	const modelRoutePrefix = "/model/:modeluuid"
-	const charmsObjectsRoutePrefix = "/:bucket/charms/:object"
+const (
+	modelRoutePrefix         = "/model/:modeluuid"
+	charmsObjectsRoutePrefix = "/:bucket/charms/:object"
+	objectsRoutePrefix       = "/:bucket/:objects/:object"
```

Comment:
I know this is not from this PR, but I'm wondering if this is OK. How can we ensure that the route `"/:bucket/:objects/:object"` is different from `"/:bucket/charms/:object"`? What I mean is that what would we do if `:objects` is `charms`? This doesn't seem correct.

Reply:
Ah, that's a typo. It should be `/:bucket/objects/:object`. I'll fix.